config/jobs/kubernetes/generated/generated.yaml

```diff
@@ -75,13 +75,6 @@ periodics:
           cpu: 1000m
           memory: 3Gi
   cluster: k8s-infra-prow-build
-  resources: &id001
```

Comment:
Rather than removing all of these resource requests can we set them at the correct level? (container)

Reply:
the containers have resource limits set as well... I have no idea what these ones were intended for... I can replace the container ones with the externally scopped ones?

Reply:
Oh, in that case removing these like you're doing seems like a good option :+1: